frontend/src/App.js

```diff
@@ -1,9 +1,56 @@
 import React from 'react'
+import { Provider } from 'react-redux'
+import {configureStore, combineReducers } from '@reduxjs/toolkit'
+import { BrowserRouter, Switch, Route } from 'react-router-dom'
+import styled from 'styled-components'
```

Comment:
Great job using styled components! 👍🏻  Perhaps you could consider using macro too, if you wish for the class names to be displayed in dev tools ⚒️